### DISCUSSION TITLE
canot find symbol  FirebaseAnalytics.Param.ITEMS

### DISCUSSION THREAD

**Discussion by praveenp-od** [invertase/react-native-firebase#5741] [General]
Hi Team,

I got the following error msg when building the Android build.  In @react-native-firebase/analytics

/node_modules/@react-native-firebase/analytics/android/src/reactnative/java/io/invertase/firebase/analytics/ReactNativeFirebaseAnalyticsModule.java:150: error: cannot find symbol
    ArrayList itemsArray = (ArrayList) bundle.getSerializable(FirebaseAnalytics.Param.ITEMS);
                                                                                                                                               ^
symbol:   variable ITEMS
location: class Param  

and 

/node_modules/@react-native-firebase/analytics/android/src/main/java/io/invertase/firebase/analytics/UniversalFirebaseAnalyticsModule.java:102: error: cannot find symbol
      FirebaseAnalytics.getInstance(getContext()).setDefaultEventParameters(parameters);

symbol:   method setDefaultEventParameters(Bundle)
location: class FirebaseAnalytics

**Comment by mikehardy** [comment]
Hi there! You have not included any of the information needed to troubleshoot this so I can't say what's going on.
I can say that we compile and test the software on every commit, and obviously it builds for us.
That implies that you have some sort of versioning issue, like perhaps you are using current react-native-firebase but you have overridden the native SDKs such that firebase-android-sdk is out of date and does not contain the correct symbols.
You can use this demonstrator to build a quick test demonstrator https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh or examine our "tests" directory to see our test app and what versions it is using.
